packages/lib/src/ui/blocks/PersonList/PersonList.tsx

```diff
@@ -0,0 +1,138 @@
+import { Block } from "../../components/Block/Block"
+import { Container } from "../../components/Container/Container"
+import styled from "@emotion/styled"
+import {
+  Header3,
+  SmallText,
+  Text,
+} from "../../components/Typography/Typography"
+import React from "react"
+import { PersonListProps } from "./personListDef"
+import { Image as ImageType } from "../../../types/data"
+
+import Circle1 from "../../assets/vectors/potatoes/circle_1.svg"
+import Circle2 from "../../assets/vectors/potatoes/circle_2.svg"
+import Circle3 from "../../assets/vectors/potatoes/circle_3.svg"
+import Circle4 from "../../assets/vectors/potatoes/circle_4.svg"
+import Circle5 from "../../assets/vectors/potatoes/circle_5.svg"
+import Circle6 from "../../assets/vectors/potatoes/circle_6.svg"
+import Circle7 from "../../assets/vectors/potatoes/circle_7.svg"
+import Circle8 from "../../assets/vectors/potatoes/circle_8.svg"
+
+import { Image } from "../../components/Image/Image"
+import { max, theme } from "../../styles/theme"
+
+const circles = [
+  Circle1,
+  Circle2,
+  Circle3,
+  Circle4,
+  Circle5,
+  Circle6,
+  Circle7,
+  Circle8,
+]
+
+export interface PersonProps {
+  name: string
+  nickname?: string
+  image: ImageType
+  subtitle?: string
+  contact?: {
+    email?: string
+    phone?: string
+  }
+}
+
+export const PersonList: React.FC<PersonListProps> = (props) => (
+  <Block>
+    <Container>
+      <PersonListContainer>
+        {props.content.map((person, index) => (
+          <Person key={index}>
+            <Picture
+              mask={circles[(index * 13 + 11) % 8].src}
+              image={person.image}
+            />
+            <Name>
+              {person.name}
+              {person.nickname && `—${person.nickname}`}
+            </Name>
+            {person.subtitle && <SubTitle>{person.subtitle}</SubTitle>}
+            {person.contact && (
+              <ContactPoints>
+                {person.contact.email && (
+                  <ContactPoint href={`mailto:${person.contact.email}`}>
+                    {person.contact.email}
+                  </ContactPoint>
+                )}
+                {person.contact.phone && (
+                  <ContactPoint href={`tel:${person.contact.phone}`}>
+                    {person.contact.phone}
+                  </ContactPoint>
+                )}
+              </ContactPoints>
+            )}
+          </Person>
+        ))}
+      </PersonListContainer>
+    </Container>
+  </Block>
+)
+
+const PersonListContainer = styled.div`
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  gap: 3em 2em;
+
+  @media ${max("l")} {
+    grid-template-columns: 1fr 1fr 1fr;
+  }
+
+  @media ${max("m")} {
+    grid-template-columns: 1fr 1fr;
+  }
+
+  @media ${max("s")} {
+    grid-template-columns: 1fr;
+  }
+`
+
+const Person = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 0.75rem;
+`
+
+const Picture = styled(Image)`
```

Comment:
Ten název je trochu zavádějící. Evokuje mi užití `<picture>` tagu. Spíš to pojmenuj sémanticky. Třeba PersonImage, nebo tak nějak.